pittsburgh/bills.py

```diff
@@ -0,0 +1,294 @@
+                responsible_person = action.pop("responsible person")
+                act = bill.add_action(**action)
+
+                if responsible_person:
+                     act.add_related_entity(responsible_person,
+                                            "person",
+                                            entity_id = _make_pseudo_id(name=responsible_person))
+
+                if action["description"] == "Referred":
+                    body_name = matter["MatterBodyName"]
+                    if body_name != "City Council":
+                        act.add_related_entity(body_name,
+                                               "organization",
+                                               entity_id = _make_pseudo_id(name=body_name))
+
+                result, votes = vote
+
+                if result:
+                    vote_event = VoteEvent(legislative_session=bill.legislative_session,
+                                           motion_text=action["description"],
+                                           organization=action["organization"],
+                                           classification=None,
+                                           start_date=action["date"],
+                                           result=result,
+                                           bill=bill)
+
+                    vote_event.add_source(legistar_web)
+                    vote_event.add_source(legistar_api + "/histories")
+
+                    for vote in votes:
+                        raw_option = vote["VoteValueName"].lower()
+                        clean_option = self.VOTE_OPTIONS.get(raw_option,
+                                                             raw_option)
+                        vote_event.vote(clean_option,
+                                        vote["VotePersonName"].strip())
+
+                    yield vote_event
+
+
+            for sponsorship in self.sponsorships(matter_id):
+                bill.add_sponsorship(**sponsorship)
+
+            for topic in self.topics(matter_id):
+                bill.add_subject(topic["MatterIndexName"].strip())
+
+            for attachment in self.attachments(matter_id):
+                if attachment["MatterAttachmentName"]:
+                    bill.add_version_link(attachment["MatterAttachmentName"],
+                                          attachment["MatterAttachmentHyperlink"],
+                                          media_type="application/pdf")
+
+            bill.extras = {"local_classification" : matter["MatterTypeName"]}
+            text = self.text(matter_id)
+
+            if text:
+                if text["MatterTextPlain"]:
+                    bill.extras["plain_text"] = text["MatterTextPlain"]
+
+                if text["MatterTextRtf"]:
+                    bill.extras["rtf_text"] = text["MatterTextRtf"].replace(u"\u0000", "")
+
+            yield bill
+
+ACTION = {"Committee Report":
+              {"ocd" : None, "order" : 0},
```

Comment:
what is "order" doing for you hear? Doesn't seem to be used in the code elsewhere?

Reply:
Ah sorry about that, that's some lazy aping of the Chicago `bills` scraper:
https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/bills.py#L235

I'm realizing, however, that I don't have the `sort_actions` function that draws on that field: 

https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/bills.py#L9-L15

Do you think `sort_actions` is worth adding, or can this scraper live without it? If the former I assume I'd want to drop it here:

https://github.com/opencivicdata/scrapers-us-municipal/pull/278/files/6eb00eeea539cf8777e85f7dc480248fd6904ee8#diff-cc73d0148d2b9fd079236552b7f45cd1R50

Reply:
you can add it later.